refs/biblio.json

```diff
@@ -766,9 +766,9 @@
             "Rich Tibbett",
             "Clarke Stevens"
         ],
-        "href": "http://people.opera.com/richt/release/specs/discovery/Overview.html",
+        "href": "https://www.w3.org/TR/2012/WD-discovery-api-20120807/",
```

Comment:
Same here as above. You can make this an alias of the auto-generated W3C reference.

The question is whether you want to alias it to the "discovery-api" or _that specific version_, in which case, alias it to "discovery-api-20120807".

You'll find all related reference listed here: http://www.specref.org/?q=discovery-api